pkg/security/ebpf/c/include/maps.h

```diff
@@ -49,6 +49,7 @@ BPF_HASH_MAP(secprofs_syscalls, u64, struct security_profile_syscalls_t, 1) // m
 BPF_HASH_MAP(auid_approvers, u32, struct event_mask_filter_t, 128)
 BPF_HASH_MAP(auid_range_approvers, u32, struct u32_range_filter_t, EVENT_MAX)
 BPF_HASH_MAP(active_flows_spin_locks, u32, struct active_flows_spin_lock_t, 1) // max entry will be overridden at runtime
+BPF_HASH_MAP(inode_file, u64, struct file_t, 32)
```

Comment:
we discussed about sizing it to 1, should we ?

Reply:
Let's keep as it is for now, I'll change to an array in a follow-up PR